ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -64,6 +77,12 @@
     public final Histogram queryMonthTimeMs;
     public final Histogram takeSnapshotTimeMs;
 
+    public final Histogram queryPartitionNameAndIdTimeMs;
+    public final Histogram storePartitionClassStatsTimeMs;
+    public final Histogram queryPartitionClassStatsTimeMs;
+    public final Histogram storeAggregatedPartitionClassStatsTimeMs;
+    public final Histogram queryAggregatedPartitionClassStatsTimeMs;
```

Comment:
We generally put metrics in their own class.  Recommend consistency.

Reply:
These metrics are for AccountStatsMysqlStore method. That's why I put it here.